src/Ginger/Resource.elm

```diff
@@ -170,15 +172,20 @@ Every resource has _one_ category, but can be part of a hierarchy of other
 categories. For example `news` is part of `text > article > news`. This function
 will always return the `Category` stored with the `ResourceWith`.
 
-_Note: there hasn't been the need to expose any of the parent categories so far,
-if there is please file an issue._
-
 -}
 getCategory : ResourceWith a -> Category
 getCategory =
     List.NonEmpty.head << .category
 
 
+{-| Get the category including its parents from a `ResourceWith` sorted
+from big to small.
```

Comment:
Get the entire category tree of a `ResourceWith` starting from the parent category and ending with the smallest child.